src/durabletask/azext_durabletask/custom.py

```diff
@@ -1,14 +1,69 @@
 # --------------------------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
-#
-# Code generated by aaz-dev-tools
 # --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
 
-# pylint: disable=too-many-lines
-# pylint: disable=too-many-statements
+import httpx
+from azure.cli.core.commands.client_factory import get_subscription_id
+from .aaz.latest.durabletask.scheduler import Show
 
-from knack.log import get_logger
 
+def list_orchestrations(cmd, resource_group_name, scheduler_name, taskhub_name, max_items=100, start_index=0):
+    # Get FQDN of the scheduler
 
-logger = get_logger(__name__)
+    scheduler = Show(cli_ctx=cmd.cli_ctx)(command_args={
+        "resource_group": resource_group_name,
+        "name": scheduler_name,
+        "subscription": get_subscription_id(cmd.cli_ctx)
+    })
+
+    endpoint = scheduler['properties']['endpoint']
+    endpoint += "/v1/taskhubs/orchestrations/query"
+    grabbed_token = f"Bearer {get_bearer_token(cmd.cli_ctx)}"
+
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization': grabbed_token,
+        'x-taskhub': taskhub_name
+    }
+
+    payload = {"filter": {}, "pagination": {"startIndex": start_index, "count": max_items}, "sort": [{"column": "LAST_UPDATED_AT", "direction": "DESCENDING_SORT"}]}
+
+    client = httpx.Client(http2=True)
+    response = client.post(endpoint, json=payload, headers=headers)
+    return response.json()
+
+
+def show_orchestration(cmd, resource_group_name, scheduler_name, taskhub_name, orchestration_id):
```

Comment:
Could we see an example of the output of this new command?

Another thing to consider is that neither of these two commands will return the orchestration's payloads (input/output/custom state), and this might be important to CLI users.  The equivalent gRPC APIs allow callers to indicate that payloads should be returned but the dashboard (the primary user of the HTTP API) prefers a more "progressive disclosure" approach where basic orchestration details are returned and then payload data is returned only with a subsequent, explicit, call.

Reply:
We might update the HTTP query API to allow opting-into returning payload data or also offer an additional CLI command to fetch the payloads for a specific orchestration.